index.js

```diff
@@ -44,9 +44,12 @@ module.exports = async function(options) {
         //
         // However, we do make sure the site is published and not in the trash,
         // to keep things intuitive for the superadmin.
-        trash: { $ne: true },
-        published: true
-      });
+        trash: { $ne: true }
+      };
+      if (published !== null) {
```

Comment:
Normally we only want to serve published sites but allow this to be explicitly bypassed. `null` is the canonical "don't care" value for otherwise boolean properties in Apostrophe.